### ISSUE TITLE
Add support for COMPRESSION argument to TDIGEST.CREATE

### ISSUE THREAD

**Issue by chayim** [redis/NRedisStack#22]
To support RedisBloom 2.4.0, we need to support the compression argument on TDIGEST.CREATE. This entereed RedisBloom as part of https://github.com/RedisBloom/RedisBloom/pull/520

**Comment by shacharPash** [comment]
There is already support for COMPRESSION argument:
```csharp
public bool Create(RedisKey key, long compression = 100)
{
        return ResponseParser.OKtoBoolean(_db.Execute(TDIGEST.CREATE, key, compression));
}
```

**Comment by shacharPash** [comment]
Do you mean to add 'COMPRESSION' before the number?
Because in the documentation two different things are written, in the example it is written without COMPRESSION and in the syntax it is written with COMPRESSION

[https://redis.io/commands/tdigest.create/](url)

**Comment by shacharPash** [comment]
Fixed in #24
